modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -345,15 +354,28 @@ def required_claimant_data_is_present
       return if api_version.upcase == 'V1'
 
       # Claimant DOB is always required if they've supplied any claimant headers
-      has_claimant_headers = claimant.birth_date.present?
+      has_claimant_headers = claimant.birth_date.present? && self.class.past?(birth_date) ? true : false
       # form data that includes a claimant is also sufficient to know it's passed the schema
-      has_claimant_data = data_attributes&.fetch('claimant', nil).present?
+      contact_info = claimant.phone_data.present? && claimant.email.present?
+      claimant_exists = claimant.form_data.present? && data_attributes&.fetch('claimant', nil).present?
+      has_claimant_data = claimant_exists && !claimant.homeless?.nil? && contact_info ? true : false
```

Comment:
No need for ternary operator here :)

Reply:
Again, check me on this, but I believe that if the incorrect data had been provided re: homeless/contact into, it would have already been caught by the model validation. I may be reading this out of context, so let me know.